src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -2113,8 +2121,13 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
         pushDesc(LayoutType::nspc, undef, true, true);
         pushDesc(LayoutType::ncsp, undef, true, true);
         canUseAclExecutor = !supportedPrimitiveDescriptors.empty();
-        if (canUseAclExecutor)
+        if (canUseAclExecutor) {
```

Comment:
You updated only `if (is_version11) {` branch, `else` body should also be fixed.

Reply:
Fixed